include/gateway_config.hrl

```diff
@@ -13,8 +13,6 @@
 -define(MINER_APPLICATION_NAME, "com.helium.Miner").
 -define(MINER_INTERFACE, "com.helium.Miner").
 -define(MINER_OBJECT(M), ?MINER_INTERFACE ++ "." ++ M).
--define(MINER_MEMBER_PUBKEY, "PubKey").
--define(MINER_MEMBER_ONBOARDING_KEY, "OnboardingKey").
```

Comment:
I'm guessing there is a corresponding PR to remove these same members from miner.

Reply:
there is: https://github.com/helium/miner/pull/330